specification/attributes/datetime_requirements.md

```diff
@@ -0,0 +1,34 @@
+# Datetime Requirements
+
+Billing data columns that provide date and time information (datetime related columns) conforming to specified rules and formatting requirements ensure clarity, accuracy and ease of interpretation for both humans and systems.
+
+All datetime related columns, defined in the FOCUS specification, MUST be represented in UTC and follow the ISO 8601 aligned formatting requirements listed below. Provider generated datetime related columns SHOULD adopt the same format requirements over time.
```

Comment:
Are we sure on providers adopting this format?  That seems like a big assumption to be giving people.

Reply:
I'll reduce this provider related expectation to _use UTC and align with ISO 8601_.

Reply:
provider-related expectations reworded to: "Provider generated date/time related columns SHOULD also be represented in UTC and conform to the ISO 8601 standard."

Reply:
I think we should have kept it as it was previously. Maybe something to revisit later? By saying should 'SHOULD', it was a recommendation not a requirement based on the fact that practitioners would benefit from some standardization of dates/times.